biblio/how_to_cite_gudhi.bib.in

```diff
@@ -127,42 +157,106 @@
 , year =        @GUDHI_VERSION_YEAR@
 }
 
-@incollection{gudhi:cython
-, author =  "Vincent Rouvreau"
-, title =   "Cython interface"
+@incollection{gudhi:PersistenceRepresentations
+, author =  "Pawel Dlotko"
+, title =   "Persistence representations"
 , publisher =  "{GUDHI Editorial Board}"
 , edition =     "{@GUDHI_VERSION@}"
 , booktitle =   "{GUDHI} User and Reference Manual"
-, url = "https://gudhi.inria.fr/python/@GUDHI_VERSION@/"
+, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group___persistence__representations.html"
 , year =        @GUDHI_VERSION_YEAR@
 }
 
-@incollection{gudhi:CoverComplex
-, author =  "Mathieu Carri\`ere"
-, title =   "Cover complex"
+@incollection{gudhi:SubSampling
+, author =  "Cl\'ement Jamin, Siargey Kachanovich"
+, title =   "Subsampling"
 , publisher =  "{GUDHI Editorial Board}"
 , edition =     "{@GUDHI_VERSION@}"
 , booktitle =   "{GUDHI} User and Reference Manual"
-, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group__cover__complex.html"
+, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group__subsampling.html"
 , year =        @GUDHI_VERSION_YEAR@
 }
 
-@incollection{gudhi:PersistenceRepresentations
-, author =  "Pawel Dlotko"
-, title =   "Persistence representations"
+@incollection{gudhi:SpatialSearching
+, author =  "Cl\'ement Jamin"
+, title =   "Spatial searching"
 , publisher =  "{GUDHI Editorial Board}"
 , edition =     "{@GUDHI_VERSION@}"
 , booktitle =   "{GUDHI} User and Reference Manual"
-, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group___persistence__representations.html"
+, url = "https://gudhi.inria.fr/doc/@GUDHI_VERSION@/group__spatial__searching.html"
 , year =        @GUDHI_VERSION_YEAR@
 }
 
-@incollection{gudhi:Collapse
-, author =  "Siddharth Pritam and Marc Glisse"
-, title =   "Edge collapse"
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Python specific gudhi modules
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+@incollection{gudhi:WeightedRipsComplex
+, author =  "Rapha\"el Tinarrage, Yuichi Ike, Masatoshi Takenouchi"
```

Comment:
I didn't pay attention, but you seem to have replaced `and` with a comma as separator in `author`. Why? AFAIK `and` is the proper separator that is understood by bibtex.

Reply:
I did that because that was sounding weird... but if that's the way to do it, ok, I was just not aware of that. I fix it

Reply:
I fixed it on #743 . Sorry about that